script/build-translations.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
```

Comment:
So one of the things that I want to do after we migrate to Polymer 2 is that we switch to using `gulp` as a build pipeline. The current approach is … pretty chaotic.